style/main.scss

```diff
@@ -61,6 +61,14 @@
     color: var(--color-prettylights-syntax-constant);
   }
 
+  &.atrule {
+    color: var(--color-prettylights-syntax-keyword);
+  }
+
+  &.punctuation {
```

Comment:
You added `key` to the options above, but didn't add a matching style. Was that intentional?

Reply:
I think the css produced purely by prism is also lacking css for this, although I could be wrong. Is it the styling looking incorrect?